Makefile

```diff
@@ -1,6 +1,7 @@
 # Package configuration
 PROJECT = borges
 COMMANDS = cli/borges
+GOFLAGS = -tags norwfs
```

Comment:
What is this build tag for?

Reply:
go-git makes use of some go-billy (filesystem) features not available when using siva files. It opens files in read write mode with locking and does file truncate. This tag compiles versions of the code that does not use those features.

https://github.com/src-d/go-git/pull/700